### PR TITLE
KIALI-1472 Move md files to asciidoc

### PR DESCRIPTION
**Pull request by aljesusg**
- Moved README  to asciidocs
- Moved contribute to asciidocs
- Moved gettingstarted to asciidocs
- Moved api content generator to asciidocs

Note: Features md files in [KIALI-1452](https://issues.jboss.org/browse/KIALI-1452) by @lucasponce

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -16,7 +16,7 @@ Or see the Hugo installation instructions here: https://gohugo.io/getting-starte
 
 On Fedora: `sudo dnf install asciidoctor`
```
Comment (lucasponce) [inline README.adoc]: Just a note, that for me (Fedora) a install of asciidoctor was not enough and I had to run a bundle install.

File: content/contribute/index.adoc [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+title: "Contribute"
+date: 2018-06-20T19:04:38+02:00
+draft: false
+menu:
+  main:
+    weight: 30
+---
+
+## Reporting issues and feature requests
```
Comment (lucasponce) [inline content/contribute/index.adoc]: asciidoc uses the character "==" instead of "##" ?

Comment (aljesusg) [reply content/contribute/index.adoc]: You can use both, I think that this only impact if you have the toc for the index content but I not tested

File: README.adoc (hunk 1 above)
Comment (aljesusg) [reply README.adoc]: Yes...We should remove it. I'll do

File: content/contribute/index.adoc [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+The Kiali team uses link:http://issues.jboss.org/browse/KIALI[JIRA at JBoss.org] as its preferred tracker. Please use it to report issues and create new feature requests (select issue type 'Bug' for bugs and 'Story' for enhancement requests).
```
Comment (jotak) [inline content/contribute/index.adoc]: [Here](https://github.com/kiali/kiali.io/pull/22/files#diff-501311b690b3b0fe5f1d56d224f2f61eR22) I'm writing:
`In general, we use the GitHub tracker for issues raised by the community, and JIRA for issues raised by the core engineering team. Both are publicly accessible.`
Which one is correct?
See also what @pilhuhn wrote, but it hasn't been merged: https://github.com/kiali/kiali/pull/369/files

Comment (lucasponce) [reply content/contribute/index.adoc]: @jotak I think this PR is to move the existing md into adoc.
After that is done we can change the wording, but this is technical task.

Comment (jotak) [reply content/contribute/index.adoc]: Ah, correct, I didn't see the removed "contribute/index.md".

**Comment by aljesusg** [comment]
Moved all to asciidoc format and fixed the readme @lucasponce @jotak . Thanks

**Pull request merged**